pages/review-barrier.tsx

```diff
@@ -0,0 +1,365 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import * as React from "react";
+import { FormProvider, useForm } from "react-hook-form";
+import { useIntl } from "react-intl";
+import Button from "../components/Button";
+import Layout from "../components/Layout";
+import Page, { LeftSection, RightSection } from "../components/Page";
+import { PaperClipIcon } from "@heroicons/react/solid";
+
+type FormValues = {
+  optionalContext: string;
+};
+
+const ReviewBarrier: React.FunctionComponent = () => {
+  const intl = useIntl();
+  const { push } = useRouter();
+  const methods = useForm<FormValues>();
+  const { handleSubmit } = methods;
+  const onSubmit = async (data: FormValues): Promise<void> => {
+    // TODO: Save barrier to cookie.
+    push(`/solutions/identify-a-solution`);
+  };
+  const bold = (msg: string): React.ReactNode => (
+    <span data-h2-font-weight="b(700)">{msg}</span>
+  );
+  const solutions = (msg: string): React.ReactNode => <a href="#">{msg}</a>;
+  return (
+    <Layout
+      title={intl.formatMessage({
+        defaultMessage:
+          "Review or edit your barrier: Concentration in a noisy workplace",
+      })}
+      headTitle={intl.formatMessage({
+        defaultMessage:
+          "Review or edit your barrier: Concentration in a noisy workplace",
+      })}
+    >
+      <div data-h2-padding="b(all, xl)">
+        <FormProvider {...methods}>
+          <form onSubmit={handleSubmit(onSubmit)}>
```

Comment:
I would remove all three of these elements since they are not needed on this page.